portal-user-datastore-rdbms/src/main/java/com/telecominfraproject/wlan/portaluser/datastore/rdbms/PortalUserRowMapper.java

```diff
@@ -33,6 +36,18 @@ public PortalUser mapRow(ResultSet rs, int rowNum) throws SQLException {
         portalUser.setPassword(rs.getString(colIdx++));
         portalUser.setRole(PortalUserRole.getById(rs.getInt(colIdx++)));
         
+        String rolesString = rs.getString(colIdx++);
+        rolesString = rolesString.replace("[", ""); // remove brackets
+        rolesString = rolesString.replace("]", ""); 
+        rolesString = rolesString.replace(" ", ""); 
+        List<String> listOfRoleIds = Arrays.asList(rolesString.split(",", -1));
+        List<PortalUserRole> listOfRoles = new ArrayList<>();
+        for (String roleIdString : listOfRoleIds) {
+        	int roleId = Integer.parseInt(roleIdString);
+        	listOfRoleIds.forEach(y -> listOfRoles.add(PortalUserRole.getById(roleId)));
```

Comment:
You probably meant listOfRoleIds.forEach(y -> listOfRoles.add(PortalUserRole.getById(Integer.parseInt(y))));
And the lines 45,46, and 48 should be removed.

Reply:
Ah, I was debugging it and it was easier with a for loop rather than a foreach, I'll change that back to foreach